modules/gallery/src/commonMain/kotlin/org/jetbrains/compose/storytale/gallery/material3/StoryContent.kt

```diff
@@ -124,22 +119,6 @@ private fun StoryPreview(
         ) {
             activeStory?.content?.invoke(activeStory)
         }
-
-        AnimatedVisibility(
-            (isSmallWidth || useEmbeddedView) && activeStory?.parameters?.isNotEmpty() == true,
-            enter = fadeIn(),
-            exit = fadeOut(),
-            modifier = Modifier.align(Alignment.BottomEnd),
-        ) {
-            SmallFloatingActionButton(
```

Comment:
This Floating Action Button was added here according to the re-design by our team's designer. It should've been removed from here.